### DISCUSSION TITLE
Mouse movement detection?

### DISCUSSION THREAD

**Discussion by cyaconi** [kovidgoyal/kitty#6264] [Q&A]
Hi,

To quickly move and resize windows, I use [Hummingbird](https://github.com/finestructure/Hummingbird) on my Mac. I simply need to hover my mouse anywhere over the window and press any of the modifier keys to resize or move it.

Kitty is the one application I use that doesn't work well with it. It appears that Hummingbird misses the app and resizes/moves the window just behind Kitty. Nevertheless, if I hover the mouse over the OS's window top bar, it works, and after that, it keeps working no matter where I hover the mouse over the terminal until I move/resize other window.

Is this behavior normal? If so, is there a way I can modify kitty.conf to prevent it?

Thank you!

**Comment by kovidgoyal** [comment]
No idea, you would need to ask the hummingbird developers.
